ui/src/main.mjs

```diff
@@ -73,13 +76,43 @@ fetch('_setup')
 
         const socket = io(setup.socketio)
 
+        let disconnected = false
+
+        function reconnect (interval = 1000) {
```

Comment:
is 1s not a bit aggressive Joe? And indefinitely? I think there should be some eventual circuit breaker where it stops trying and puts the ball into the users hands (i.e. click a retry button)

Thinking about the issue we had recently where an editor was left open and it kept trying to reach a deleted/suspended instance indefinitely (didnt it cause an alert?)

In these situations, it is V common to see an exponential back-off and a circuit breaker.

Reply:
It's a fair point, happy to go smaller and exponentially increase until limit is reached

Reply:
Have updated